drivers/sensor/adt7420/adt7420.c

```diff
@@ -11,8 +11,12 @@
 #include <kernel.h>
 #include <sensor.h>
 #include <misc/__assert.h>
+#include <logging/log.h>
```

Comment:
move that below and after LOG_LEVEL is defined, otherwise you will get the default log level and not the one you are specifying below.

Reply:
not needed any more. Now log level is resolved only when used so order does not matter.